lsp-origami.el

```diff
@@ -63,12 +63,13 @@
   :group 'lsp-origami
   :global nil
   (cond
-   (lsp-origami-mode
-    (setq-local origami-fold-style 'lsp-mode))
+   ((and (bound-and-true-p lsp-mode)
+         (lsp--capability "foldingRangeProvider"))
```

Comment:
I added it to `lsp-after-open-hook` and simply added in the requirement of `origami-mode` being `t`. Does that work?

Reply:
It will work as long as origami-mode is enabled before lsp-mode. Can you add that as a comment? Alternatively, we might add the same handler hook to origami-mode-hook in the readme.

Reply:
```emacs-lisp
(add-hook 'lsp-after-open-hook #'lsp-origami-mode)
(add-hook 'origami-mode-hook #'lsp-origami-mode)
```
Are you suggesting to put it on both hooks? It seems like that would still allow the possibility of the problem you suggested before where the `lsp--capability` returns nil because the server hasn't yet initialized.

Reply:
> ```emacs-lisp
> (add-hook 'lsp-after-open-hook #'lsp-origami-mode)
> (add-hook 'origami-mode-hook #'lsp-origami-mode)
> ```
> 
> Are you suggesting to put it on both hooks? It seems like that would still allow the possibility of the problem you suggested before where the `lsp--capability` returns nil because the server hasn't yet initialized.

Yes. AFACS lsp-after-open-hook is called after the capabilities are populated, so we are fine.

Reply:
Actually, I think that we should add another method e. g. `lsp-origami-try-enable` which contains the logic that is currently in lsp-origami-mode because if both hooks run after each other it looks like the second time they won't run.